tomb

```diff
@@ -654,6 +654,7 @@ usage() {
     _print " -f     force operation (i.e. even if swap is active)"
     _print " -g     use a GnuPG key to encrypt a tomb key"
     _print " -r     provide GnuPG recipients (separated by coma)"
```

Comment:
How long should the coma be, to make sure it acts as a separator :P ;)

Dunno if it was done as a joke, but imo it doesn't fit into the undertaker theme, so it should be the comma?